### PR TITLE
Fix several typos, omissions I've noticed in the documentation.

### PR DESCRIPTION
**Pull request by manuelCarlos**
This PR fixes several typos, omissions I've noticed in the documentation. :rocket:

## Checklist
- [X ] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.

Did not add a changelog since this contribution likely does not count as a  notable change 😉

### CODE REVIEW

**Review by LucianoPAlmeida — APPROVED** [review]
LGTM! Thank you =]

**Review by guykogus — APPROVED** [review]
Awesome. Did you use a script or tool to find/fix all these?

**Comment by LucianoPAlmeida** [comment]
@manuelCarlos Does your branch is updated with origin/master? If not can you rebase? 
I'm trying to see what is happening with the CI :)

**Comment by manuelCarlos** [comment]
> Awesome. Did you use a script or tool to find/fix all these?

Hey, thanks. It's a pleasure to help a little bit!
Am using a personal custom tool that I've been playing with, trying to (mostly failing) to improve 😄 . It's using SourceKit paired with some very crude ML. 
I thought I'd try it on this repo because the documentation is actually already in very good shape and is very consistent, just to see how it would go.

**Comment by manuelCarlos** [comment]
> @manuelCarlos Does your branch is updated with origin/master? If not can you rebase?
> I'm trying to see what is happening with the CI :)

Yes, I think  this branch is up-to-date with origin/master 👍🏻

**Comment by LucianoPAlmeida** [comment]
> Yes, I think this branch is up-to-date with origin/master 👍🏻

Thanks, we are going to check what is going on with the CI so we can merge this ASAP :)

**Comment by guykogus** [comment]
@manuelCarlos can you please run `brew bundle` and commit the updated `Brewfile.lock.json` file?

**Comment by manuelCarlos** [comment]
Well, CI is still red. Please let me know if there's anything I can help on my side.

File: Sources/SwifterSwift/CoreGraphics/CGPointExtensions.swift [hunk 1]
```diff
@@ -62,7 +62,7 @@ public extension CGPoint {
     ///     // point1 = CGPoint(x: 40, y: 40)
     ///
     /// - Parameters:
-    ///   - lhs: self
+    ///   - lhs: self.
```
Comment (guykogus) [inline Sources/SwifterSwift/CoreGraphics/CGPointExtensions.swift:65]: :\

Comment (manuelCarlos) [reply Sources/SwifterSwift/CoreGraphics/CGPointExtensions.swift:65]: #consistency trumps aesthetics

Comment (manuelCarlos) [reply Sources/SwifterSwift/CoreGraphics/CGPointExtensions.swift:65]: How about 
```swift
- lhs: `self`.
```

**Comment by LucianoPAlmeida** [comment]
@manuelCarlos Can you please revert the brew file  changes?
#938 fixed the issue so no need for that =]

**Comment by manuelCarlos** [comment]
> @manuelCarlos Can you please revert the brew file changes?
> #938 fixed the issue so no need for that =]

Done ✅

**Review by LucianoPAlmeida — APPROVED** [review]

**Review by guykogus — APPROVED** [review]

**Comment by SD10** [comment]
Thank you for contributing to SwifterSwift! I've invited you to join the SwifterSwift GitHub organization - no pressure to accept! If you'd like more information on what that means, check out our [contributing guidelines](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md). Feel free to reach out if you have any questions! 😃

**Pull request merged**